data/kaushalnitb.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Kaushal Kishor",
```

Comment:
hey, @kaushalnitb you mistakenly added an extra code. kindly delete these from lines 1 to 6. if you want to add clear `json` code from scratch you can check the [official doc](https://linkfree.eddiehub.io/docs/full-profile-example) 

```diff
{
  "name": "Kaushal Kishor",
  "displayStatsPublic": true,
  "type": "personal",
  "bio": "I am a DevOps engineer",
  "links": [
```